src/components/WebexMeetingInfo/WebexMeetingInfo.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import jsonData from '../../data';
+import {WebexJSONAdapter} from '../../adapters';
+import {WebexMeetingInfo, WebexDataProvider} from '../';
+
+// Setup for the stories
+const stories = storiesOf('Webex Meeting Info', module);
+const webexAdapter = new WebexJSONAdapter(jsonData);
+const wrapperStyle = {height: '150px', width: '500px', border: '1px solid black'};
```

Comment:
I put a wrapper around the elements for the storybook so that it would display with alignment well.

Reply:
I've been thinking about how to standardized this for "shapeless" components, I was leaning towards shadows, but this is good for now